### DISCUSSION TITLE
HW 2 Part 3.1

### DISCUSSION THREAD

**Discussion by deena101** [cooperrc/computational-mechanics#48] [Q&A]
Is this the data set everyone else got? How do I call for EUR v AMR if it doesn't show up?

![image](https://user-images.githubusercontent.com/106576120/173194879-28010dbf-32d1-4088-9566-9c7e95a90824.png)

**Comment by FishMacaron** [comment]
I'm not really sure what you did to get that, but I personally did this to retrieve the info for all of Europe (did the same for the US):
> EUR = energy[energy['Entity']=='Europe']

And then I retrieved the data of the years and energy consumed from EUR and USA (the variable name I used for the US) in a similar way before using those variables as my x and y respectively when plotting.

But I'm pretty sure from the data you pulled you can do something similar, I'm sure the info for Europe and the US are in there, it may not just be printing the whole thing? I'm not really sure though since I can't see your code.

**Reply by deena101** [reply]
Thanks! That's exactly what it was, it just cut the data short.
